fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/identifiers/FedoraId.java

```diff
@@ -518,11 +518,7 @@ private String processIdentifier() {
         }
         if (processID.contains(FCR_VERSIONS)) {
             final String[] versionSplits = split(processID, FCR_VERSIONS);
-            if (versionSplits.length > 2) {
```

Comment:
did you conclude this was unreachable?

Reply:
Yeah, in the private constructor we call this method [after we call](https://github.com/fcrepo/fcrepo/blob/main/fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/identifiers/FedoraId.java#L113-L114) `checkForInvalidPath` which also checks for 2 instances of any of the special path elements (`fcr:acl`, `fcr:versions`, `fcr:tombstone` and `fcr:metadata`). So this doesn't get hit.